docs/docker.rst

```diff
@@ -0,0 +1,86 @@
+==================
+Docker Development
+==================
+
+First, install `Docker https://docs.docker.com/installation/`_ and `Compose https://docs.docker.com/compose/install/`_. Then clone bedrock locally with::
+
+    $ git clone --recursive git://github.com/mozilla/bedrock.git
+    $ cd bedrock
+
+**(Make sure you use --recursive so that legal-docs are included)**
+
+To work with locales other than en-US, you will also need to clone the locales from their subversion repository::
+
+    $ git svn clone https://svn.mozilla.org/projects/mozilla.com/trunk/locales/ locale
+    # or
+    $ svn checkout https://svn.mozilla.org/projects/mozilla.com/trunk/locales/ locale
+
+OS X
+----
+
+Launch a dev server container in a boot2docker VM::
+
+    $ bin/b2d-dev
```

Comment:
After this step, I sat at this `Attaching to bedrock_b2d_1` message for a long time. Maybe should add a bit to the docs that says something like, "If everything works, you'll see `Attaching to bedrock_b2d_1`. then you can ..."